src/logic/logic.shaii.ts

```diff
@@ -40,19 +40,26 @@ export function timeSince(date: number) {
 export function durationToMilliseconds(duration: string): string {
   const [digits, abbr] = duration.match(/\d{1,3}|\D/g) as string[];
   switch (abbr) {
-    case "s" || "S":
+    case "s":

```

Comment:
why was this changed

Reply:
Because it's not Python it does not work that way xd